ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -54,6 +54,8 @@ import scala.concurrent.{Await, ExecutionContext, Future}
 import scala.util.Try
 
 object SandboxServer {
+  type Port = Int
```

Comment:
In general, I would recommend _not_ using type aliases, unless there's a strong motivation to do it. They add one level of indirection when reasoning about types without providing any additional type safety. The aliased types can be used interchangeably, which can lead to some unwanted :spaghetti:.

```scala
scala> type Port = Int
defined type alias Port

scala> val a: Port = 1
a: Port = 1

scala> val b: Int = 2
b: Int = 2

scala> def sum(n: Int, m: Int): Int = n + m
sum: (n: Int, m: Int)Int

scala> def sumPort(n: Port, m: Port): Int = n + m
sumPort: (n: Port, m: Port)Int

scala> def sumPort(n: Port, m: Port): Port = n + m
sumPort: (n: Port, m: Port)Port

scala> sum(1, 2)
res0: Int = 3

scala> sumPort(1, 2)
res1: Port = 3
```

I would suggest you either simply use `Int` or you make `Port` perform some useful check, e.g.:

```scala
final implicit class Port(val n: Int) extends AnyVal {
  def isValid: Boolean = n >= 0 && n <= 65535
}
```
But I don't really see a true advantage in doing with this right now.

Reply:
Happy to turn this into a proper class. The type signature for `SandboxState#reset` got confusing when it was just talking about `Int`.

Reply:
Gonna revert to just using `Int` with a comment for now.